### PR TITLE
[WIP] asu: update source to v0.2.4 and Makfile url/email

### PR DESCRIPTION
**Pull request by aparcar**
Maintainer: me 

* Support `revision`checks for snapshots. Before a  snapshots running device would be updated every time
* Add development `config.yml`
* Use `odbc.ini` instead of adding settings to `config.yml`
* Rework Ansible

### CODE REVIEW

File: devel/asu/files/asu.init [hunk 1]
```diff
@@ -23,6 +23,8 @@ start_service() {
       [ $((i++)) -gt 30 ] && return 1
     done
 
+    cp /etc/asu/odbc.ini /var/run/asu/.odbc.ini
```
Comment (aparcar) [inline devel/asu/files/asu.init]: @dangowrt is the dest folder already created at this point?

Comment (dangowrt) [reply devel/asu/files/asu.init]: `/var` is tmpfs, so you do need to create the folder before using `mkdir -p`.

**Comment by dangowrt** [comment]
I don't get the

> Rework Ansible

part in the commit message. Is this related to OpenWrt at all?

**Comment by BKPepe** [comment]
ping @aparcar

**Comment by diizzyy** [comment]
Closing due to timeout

**Pull request closed without merging**